README.md

```diff
@@ -19,7 +19,7 @@ Via Composer:
 $ composer require leeovery/wordpress-to-laravel
 ```
 
-After adding the package, add the ServiceProvider to the providers array in config/app.php:
+This package is auto-discoverable with Laravel, but if you want to be anal go ahead and add the ServiceProvider to the providers array in config/app.php:
```

Comment:
Whilst I appreciate the humour - can you remove the anal comment!! lol!!

Reply:
This is actually in you master branch, as i submited the PR to the develop branch it's showing up in the diff :)

Reply:
haha!!!! Maybe I wrote it once upon a time then. I honestly dont recall. Its been a long time since I used this pckage.